CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 - Enable translation of transparency label in map settings.
 - Remove unnecessary Italian text in publish confirmation modal.
 - Add status submit message in all languages. 
+- Display dash for empty coordinate values.
```

Comment:
REQ: Dieser Eintrag ist überflüssig und kann entfernt werden, da die Releasenotes für diesen Fix bereits in einem der vorherigen PRs eingetragen wurden. Siehe weiter oben → `Empty text or numeric fields are always displayed as `-` in the detail sections of the viewer mode.`

Reply:
REQ: Aktuellen `main`-Branch in diesen Branch mergen.

Reply:
Beide Punkte erledigt.